src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -21,34 +23,60 @@ type TableRowData = RowData & {
 
 // Columns
 
+function cellTrivialTooltip(displayValue: string) {
+  return <Tooltip title={displayValue}>{displayValue}</Tooltip>;
+}
+
+function cellNullTooltip() {
+  return (
+    <Tooltip title="NULL">
+      <i>NULL</i>
+    </Tooltip>
+  );
+}
+
+function cellTooltip(tooltip: string, displayValue: string) {
+  return <Tooltip title={tooltip}>{displayValue}</Tooltip>;
+}
+
 function renderValue(v: Value) {
+  // we must disable the default `ellipsis: {showTitle: true}` tooltip, so we're serving
```

Comment:
I don't understand this comment, since `showTitle` is set to `false` later on.

Reply:
The way the comment was phrased, I understood that we're serving tooltips everywhere in order to disable the default behavior, which is exactly the inverse of what is happening. Maybe rephrase it like this:
"Because we later disable default tooltips, we must serve a trivial `Tooltip` node everywhere."

Reply:
yup, this is more or less the vein of the new wording. Thx!